pyproject.toml

```diff
@@ -53,10 +54,21 @@ httomo = "httomo.cli:main"
 repository = "https://github.com/DiamondLightSource/HTTomo"
 
 [project.optional-dependencies]
-tomopy = [ "tomopy" ]
-httomolib = [ "httomolib" ]
-httomolibgpu = ["httomolibgpu", "cupy"]
-dev = [
+cpu = ["tomopy" ]
+gpu = ["httomolibgpu"]
```

Comment:
```suggestion
gpu = ["httomolibgpu", "cupy"]
```

Reply:
I think httomolibgpu already specifies cupy as a dependency https://github.com/DiamondLightSource/httomolibgpu/blob/main/pyproject.toml#L41, so I didn't know if this was necessary or not; do you know if `pip` in combination with dependencies in `pyproject.toml` figures out "dependencies of dependencies" nicely?

Reply:
yep, right, it should be in the environment already then.